.github/workflows/download_test.yml

```
@@ -169,29 +174,28 @@ jobs:
               open_jtalk_dic_utf_8-1.11
     runs-on: ${{ matrix.os }}
     name: ${{ matrix.name }}-${{ matrix.os }}
-    env:
-      EXPECTED_VOICEVOX_CORE_VERSION: latest
-      # See https://github.com/VOICEVOX/voicevox_core/issues/310
-      #EXPECTED_VOICEVOX_CORE_VERSION: ${{ matrix.expected_version || 'latest' }}
     steps:
       - uses: actions/checkout@v3
       - name: Set up Rust
-        if: ${{ startsWith(matrix.download_command, 'cargo ') }}
         uses: ./.github/actions/rust-toolchain-from-file
+      - name: Get prerelease latest version
+        if: ${{ env.VERSION == 'prerelease-latest' }}
+        run: |
+          VERSION=$(
+            curl -s https://api.github.com/repos/VOICEVOX/voicevox_core/releases \
```

十分ご存知とは思っていますがcurlみたいなのはコケまくるので、`-S`と`-f`を付ければエラーがましになると思います (今のままだとjqのエラーとして表れると思います)

```suggestion
            curl -sSf https://api.github.com/repos/VOICEVOX/voicevox_core/releases \
```

```console
 -S, --show-error      Show error even when -s is used
```

```console
 -f, --fail            Fail fast with no output on HTTP errors
```

---

雰囲気で使ってました。。ありがとうございます！！